Unable to disable devmapper tests

I'm trying to update `containerd` in LinuxKit to v1.3.2 but running into issues with the recently added devmapper snapshot support (probably a combination of Alpine utilities and kernel config, see https://github.com/linuxkit/linuxkit/pull/3426#issuecomment-571801117  for details).

For LinuxKit we are not really that interested in the devmapper snapshotter but would like to continue to run the containerd root tests. I can compile `containerd` with `BUILDTAGS="no_devmapper"` but `root-test` would still try to run the devmapper tests.

On my Ubuntu dev machine (where `make root-test` works:
```
root@rlin01:~/src/go/src/github.com/containerd/containerd# make root-test BUILDTAGS="no_devmapper"
+ root-test
ok      github.com/containerd/containerd/archive        4.578s
ok      github.com/containerd/containerd/metadata       31.426s
ok      github.com/containerd/containerd/mount  2.860s
ok      github.com/containerd/containerd/snapshots/btrfs        31.901s
time="2020-01-07T22:04:08Z" level=info msg="initializing pool device \"test-pool-device-258912702\""
time="2020-01-07T22:04:08Z" level=info msg="using dmsetup:\nLibrary version:   1.02.145 (2017-11-03)\nDriver version:    4.37.0"
--- FAIL: TestPoolDevice (1.36s)
    --- FAIL: TestPoolDevice/CreateSnapshotDevice (0.19s)
        assert.go:122: assertion failed: error is not nil: 2 errors occurred:
                        * failed to activate new thin device "snap-1" (dev: 3): 1 error occurred:
                        * device or resource busy


                        * failed to delete device "snap-1" (dev id: 3): 1 error occurred:
                        * no data available



                : failed to create snapshot from 'thin-1' volume
        assert.go:122: assertion failed: error is not nil: failed to mount /tmp/containerd-mount726925173: no such file or directory
```

This is with current tip of master:
```
0d276ece0e28 ("Merge pull request #3932 from AkihiroSuda/ovl-test-opts")
```

--- a/Makefile
+++ b/Makefile
@@ -81,6 +81,11 @@ TEST_REQUIRES_ROOT_PACKAGES=$(filter \
 	done | sort -u) \
     )
 
+ifdef SKIPTESTS
+    PACKAGES:=$(filter-out ${SKIPTESTS},${PACKAGES})
+    TEST_REQUIRES_ROOT_PACKAGES:=$(filter-out ${SKIPTESTS},${TEST_REQUIRES_ROOT_PACKAGES})
+endif
+
 # Project binaries.
 COMMANDS=ctr containerd containerd-stress
 MANPAGES=ctr.1 containerd.1 containerd-config.1 containerd-config.toml.5

